docs/user-guide/src/version_support.md

```diff
@@ -3,43 +3,40 @@
 The Cluster API Provider Metal3 (CAPM3) team maintains the following branches for CAPM3 for different API versions.
 
 - CAPM3
-    - main
-        - v1beta1
-    - release-1.2
-        - v1beta1
-    - release-1.1
-        - v1beta1
+  - main
+    - v1beta1
+  - release-1.2
```

Comment:
[release-1.3](https://github.com/metal3-io/cluster-api-provider-metal3/tree/release-1.3) is missing